pipeline.py

```diff
@@ -72,31 +90,89 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        for article_item in self.corpus_manager.get_articles().values():
+            tokens = self._process(article_item.get_raw_text())
+            tokens_for_article = []
+            single_tagged_tokens = []
+            multiple_tagged_tokens = []
+            for token in tokens:
+                tokens_for_article.append(token.get_cleaned())
+                single_tagged_tokens.append(token.get_single_tagged())
+                multiple_tagged_tokens.append(token.get_multiple_tagged())
+            article_item.save_as(' '.join(tokens_for_article), kind='cleaned')
+            article_item.save_as(' '.join(single_tagged_tokens), kind='single_tagged')
+            article_item.save_as(' '.join(multiple_tagged_tokens), kind='multiple_tagged')
 
     def _process(self, raw_text: str):
         """
         Processes each token and creates MorphToken class instance
         """
-        pass
+        article_text = raw_text.replace('-\n', '')
+        for symbol in ['\n', '\r']:
+            article_text = article_text.replace(symbol, ' ')
```

Comment:
I do not see a reason to make such a preprocessing. Mystem can accept "dirty" text (with all the punctuations and other staff) and then it will process it accordingly, you only need to properly parse its output report